doc/contributing.rst

```diff
@@ -7,23 +7,36 @@ call functions in the Stan C++ library's ``stan::services`` namespace.
 **httpstan** was originally developed as a "backend" for a Stan interface
 written in Python, PyStan_.
 
-httpstan emerged out of a rewrite of an existing piece of software (PyStan)
-which was difficult to maintain. httpstan aims to be maintainable. Code
-contributions should be readable and easy to understand.
+Goals:
+
+- Provide access to frequently-used functions in Stan C++ library's ``stan::services`` namespace.
```

Comment:
Just a comment/question, no need to change anything.


`Provide access to frequently-used functions`

I like the wording here, but why we don't try to add access to all functions in `stan::services`?

Reply:
Yep, read what is in there,  `frequently-used` is good.